README.md

```diff
@@ -110,8 +133,25 @@ There is, however, a bug that breaks the JSON when the playbook execution fails
 }
 ```
 
-##### Windows Hosts
+#### Relative path to playbooks within StackStorm workflows
+Current working directory (`CWD`) defaults to pack dir you're invoking Ansible pack actions from.
+That means if you're calling `ansible.playbook` from the `custom.workflow`, then you can use relative path to playbooks you'd ship with the `custom` pack (infra-as-code, yeah).
+```
+version: '2.0'
+custom.workflow:
+  description: A sample workflow that demonstrates how to use relative paths to playbooks shipped with pack.
+  type: direct
+  tasks:
+    a:
+      action: ansible.playbook
+      input:
+        # 'ansible_play.yml' is part of the 'custom' pack
+        playbook: "ansible_play.yml"
```

Comment:
@nmaludy Check the example & README description.

The fact that you asked means it's better to provide an example + readme.
Now I think it clarifies it better.

Reply:
Also I hope _now_ it does exactly what @cognifloyd was proposing in #9